emulate-browsers/chrome-80/data/config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I like the data dir, but I wonder if the engine and maybe config should be at top level?

Reply:
Maybe, but it's a simple copy and paste from Shapeshifter right now.

Reply:
Oh, you're driving the engine from shapeshifter now. Gotcha.. I had that coming first from the emulator before. I'm not sure about this one. I guess it's fine this way. Just feels a little buried what engine is underneath, if that matters.

Reply:
I agree. One of the next items on my plate is to create all the new emulators... I think that's going to shift the emulator structure a bit... might be a good time to change it then... right now it's nice to just copy and past the dir and know all the data files were copied.